assessment/libs/matrix.html

```diff
@@ -233,6 +235,25 @@ <h3><a name="matrixFormMatrixFromEigValEigVec">matrixFormMatrixFromEigValEigVec<
 original matrix<br>
 returns the matrix PAP^-1<br>
 </p>
+
+<h3><a name="matrixFromEigenvals">matrixFromEigenvals</a></h3>
+<p>matrixFromEigenvals(eigenvalues)<br>
+Generates a matrix and random eigenvectors, where both have integer values, based on a 
+set of eigenvalues.<br>
+eigenvalues: An array of eigenvalues.  If complex, give one of the conjugate pair as [real,imag].<br>
+Returns array(matrix, eigenvectors), where matrix is in the usual format, and eigenvectors is 
+an array of eigenvectors, where each element of the array is an array of the elements of that 
+eigenvector. If an element is complex, it is returned as [real,imag].  If an eigenvector has 
+complex elements, only one of the conjugate pair is returned.
+</p>
+
+<h3><a name="matrixFormatEigenvecs">matrixFormatEigenvecs</a></h3>
+<p>matrixFormatEigenvecs(eigenvectors)<br>
```

Comment:
It'd be helpful to add more information about the format of eigenvectors.

Reply:
I suppose.  I think I mentioned that it would take the output of the `matrixFromEigenvals` function, and I'm imagining that'll be the main use of this, so I didn't bother to go into details, but I suppose it's better to be explicit.